src/main/java/seedu/address/model/Entity/Participant.java

```diff
@@ -20,15 +19,6 @@ public Participant(Name name, Email email, Phone phone, Id id) {
         super(id, name);
```

Comment:
This might just be nitpicking but you think you could reorder the params so that id and name come first? I think that makes more sense...

Reply:
Nope thats a good point, will do